nalgebra-sparse/src/io/matrix_market.rs

```diff
@@ -739,8 +739,10 @@ struct MatrixMarketParser;
 /// --------
 /// ```
 /// use nalgebra_sparse::io::load_coo_from_matrix_market_file;
-/// // Use e.g. `f64` for floating-point matrices
-/// let matrix = load_coo_from_matrix_market_file::<i32>("path/to/matrix.mtx")?;
+/// // Use e.g. `i32` for integer matrices
+/// let matrix = load_coo_from_matrix_market_file::<i32,_>("path/to/matrix.mtx");
+/// // extract the real matrix here by
+/// // let matrix = matrix.unwrap();
```

Comment:
Right, apparently I hadn't run the doc tests!

I'd recommend changing this to (can remove all the comments):
```rust
let matrix: CooMatrix<i32> = load_coo_from_matrix_market_file("path/to/matrix.mtx").unwrap();
```